ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -517,6 +525,18 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		0ADBD5822915B9C100C8BC06 /* Divider */ = {
+			isa = PBXGroup;
+			children = (
+				0ADBD5862915B9C100C8BC06 /* Divider.swift */,
+				0ADBD5852915B9C100C8BC06 /* DividerModifiers.swift */,
+				0ADBD5832915B9C100C8BC06 /* DividerTokenSet.swift */,
+				0ADBD5842915B9C100C8BC06 /* MSFDivider.swift */,
+			);
+			name = Divider;
+			path = FluentUI/Divider;
+			sourceTree = SOURCE_ROOT;
```

Comment:
interesting why this is set to sourceRoot? i would have thought path is "Diver" and sourcetree is "<group>
 like others.

Reply:
Yea not sure but I changed it with the next iteration!